### PR TITLE
add engine.io-parser_rhel_7.6 build script

### PR DESCRIPTION
**Pull request by sudeepraj2**

### CODE REVIEW

**Comment by seth-priya** [comment]
@amitsadaphule please review, thanks

**Comment by amitsadaphule** [comment]
Yes @seth-priya, I could see some obvious issues in the script and have provided some review comments to Sudeep. Once he pushes the updated script, I'll do the final review.

File: e/engine-io-parser/engine.io-parser_rhel_7.6.sh [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+git clone https://github.com/rase-/zuul-ngrok.git && cd zuul-ngrok
+npm install --save file:../ngrok
+
+#3. Building engine.io-parser
+cd ..
+git clone https://github.com/socketio/engine.io-parser.git && cd engine.io-parser/
+git checkout $PACKAGE_VERSION
+npm install --save file:../zuul-ngrok
+
+npm -y install --save-dev mocha
```
Comment (amitsadaphule) [inline e/engine-io-parser/engine.io-parser_rhel_7.6.sh:68]: Can we not add this before [npm install --save file:../zuul-ngrok](https://github.com/ppc64le/build-scripts/pull/742/files#diff-0b86dd4c95c958ec0f5306611d59aee00f67c42369084035a10493900d81a565R66) and get rid of the extra install command below?

File: e/engine-io-parser/engine.io-parser_rhel_7.6.sh [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+npm install
+
+make test
```
Comment (amitsadaphule) [inline e/engine-io-parser/engine.io-parser_rhel_7.6.sh:71]: Can you please change this to `npm test` just for consistency?

**Comment by amitsadaphule** [comment]
Can you please change the directory name to `engine.io-parser` (the package name)?

**Review by amitsadaphule — CHANGES_REQUESTED** [review]

File: e/engine-io-parser/engine.io-parser_rhel_7.6.sh (hunk 1 above)
Comment (amitsadaphule) [reply e/engine-io-parser/engine.io-parser_rhel_7.6.sh:68]: Or maybe club the --save and --save-dev options in one command, if that works?

Comment (sudeepraj2) [reply e/engine-io-parser/engine.io-parser_rhel_7.6.sh:68]: We cannot club the --save and --save-dev options in one command, it ran into issues.
We also need npm install at the end, without doing that build is getting failed on node_modules having multiple dependency.

File: e/engine-io-parser/engine.io-parser_rhel_7.6.sh (hunk 2 above)
Comment (sudeepraj2) [reply e/engine-io-parser/engine.io-parser_rhel_7.6.sh]: Updated the code, now using npm test.

**Review by amitsadaphule — APPROVED** [review]
LGTM

**Comment by amitsadaphule** [comment]
@sudeepraj2, it seems that you've kept both the directories in the commit. Please remove the other one.

**Pull request merged**